pepper-apis/src/main/resources/db-changes/schema/DDP-4852-queued-event-add-column.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sampath" id="20200815-queued-event-add-column">
+        <addColumn tableName="queued_event">
+            <column name="study_activity_id" type="bigint(20)">
```

Comment:
Adding activity_id to queued_event seems awkward and out-of-place. And more prone to error since someone can just add an arbitrary activity_id even though the event configuration has a specific activity specific. I don't think we need to add this id. The `EventConfigurationDto` should have the `activityInstanceCreationStudyActivityId` that you can use.

Reply:
oops. How did I miss that. That would have saved me good time modifying unnecessary classes. Thank you.